library/minecraft_server_file.py

```diff
@@ -59,7 +59,9 @@ def __init__(self, module):
         self.path = os.path.expanduser(module.params['path'])
         self.owner = module.params['owner']
         self.group = module.params['group']
-        self.mode = module.params['mode'] if module.params['mode'] else 0644
+        # Python version agnostic way of setting permissions 0644
+        perm_0644 = stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH
```

Comment:
We require Python 2.7 so you can just write `0o644`. Likely more understand than these flags.

Reply:
Oh, I didn't realize that format was compatible with both major versions. I agree it will be much more readable. I'll update the PR this evening.